src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -668,6 +621,131 @@ case class Dependencies(toMap: Map[MavenGroup, Map[ArtifactOrProject, ProjectRec
 object Dependencies {
   def empty: Dependencies = Dependencies(Map.empty[MavenGroup, Map[ArtifactOrProject, ProjectRecord]])
 
+  /**
+   * Combine as many ProjectRecords as possible into a result
+   */
+  def normalize(candidates0: List[(ArtifactOrProject, ProjectRecord)]): List[(ArtifactOrProject, ProjectRecord)] = {
+    type AP= (ArtifactOrProject, ProjectRecord)
```

Comment:
Random style nit, but i presume should be a space after the `AP` ?

Reply:
I'll fix that. Thanks for catching!